### ISSUE TITLE
VoiceOver on macOS does not announce fieldset description from aria-describedby when focussing inputs

### ISSUE THREAD

**Issue by 36degrees** [alphagov/reported-bugs#11]
**Upstream bug:** https://bugs.webkit.org/show_bug.cgi?id=185246
**Date:** 2018-05-03
**Reported by:** Oliver Byford
**Related to:** https://github.com/alphagov/govuk-frontend/pull/681

---

### TEST CASE:

Given the following HTML within a valid HTML5 document:

```
  <form>
    <fieldset aria-describedby="hint error">
      <legend>
        <h1>Have you changed your name?</h1>
      </legend>

      <span id="hint">This includes changing your last name or spelling your name differently.</span>
      <span id="error">Choose yes or no</span>

      <input name="changed_name" id="yes" type="radio" value="yes" >
      <label for="yes">Yes</label>

      <input name="changed_name" id="no" type="radio" value="no">
      <label for="no">No</label>

    </fieldset>
  </form>
```

### EXPECTED BEHAVIOUR:

When focussing the first radio input ("Yes") in the fieldset, Voiceover should include the text content of any element(s) associated with the containing fieldset (using aria-describedby):

> "Yes, radio button, 1 of 2, Have you changed your name? This includes changing your last name or spelling your name differently. Choose yes or no.
>
> You are currently on a radio button, 1 of 2, inside of a group. To select this option press Control-Option-Space"

This is the behaviour currently exhibited by iOS 10/11, and would be consistent with other assistive technologies (as tested with JAWS 18, NVDA).

### ACTUAL BEHAVIOUR:

When focussing the first radio input in a fieldset, Voiceover announces:

>"Yes, radio button, 1 of 2, Have you changed your name?
> 
> You are currently on a radio button, 1 of 2, inside of a group. To select this option press Control-Option-Space"

As tested in Safari 11 and Safari Technology Preview 11.2 on macOS 10.13.4.

**Comment by 36degrees** [comment]
This was fixed in Safari technology preview 59:
https://developer.apple.com/safari/technology-preview/release-notes/#r59
